sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/DataLakeTestBase.java

```diff
@@ -205,32 +202,22 @@ protected void afterTest() {
     }
 
     protected DataLakeServiceClient getOAuthServiceClient() {
+        return getOAuthServiceClientBuilder().buildClient();
+    }
+
+    protected DataLakeServiceClientBuilder getOAuthServiceClientBuilder() {
         DataLakeServiceClientBuilder builder = new DataLakeServiceClientBuilder()
             .endpoint(ENVIRONMENT.getDataLakeAccount().getDataLakeEndpoint());
 
-        instrument(builder);
-
-        Configuration configuration = Configuration.getGlobalConfiguration();
-        if (!interceptorManager.isPlaybackMode()) {
-            // Determine whether to use the environment credential based on the shared configurations or to use the
-            // credential based on resource deployment.
-            if (!CoreUtils.isNullOrEmpty(configuration.get(Configuration.PROPERTY_AZURE_TENANT_ID))
-                && !CoreUtils.isNullOrEmpty(configuration.get(Configuration.PROPERTY_AZURE_CLIENT_ID))
-                && !CoreUtils.isNullOrEmpty(configuration.get(Configuration.PROPERTY_AZURE_CLIENT_SECRET))) {
-                // AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET
-                return builder.credential(new EnvironmentCredentialBuilder().build()).buildClient();
-            } else {
-                // STORAGE_TENANT_ID, STORAGE_CLIENT_ID, STORAGE_CLIENT_SECRET
-                return builder.credential(new ClientSecretCredentialBuilder()
-                    .tenantId(configuration.get("STORAGE_TENANT_ID"))
-                    .clientId(configuration.get("STORAGE_CLIENT_ID"))
-                    .clientSecret(configuration.get("STORAGE_CLIENT_SECRET"))
-                    .build()).buildClient();
-            }
+        if (ENVIRONMENT.getTestMode() != TestMode.PLAYBACK) {
```

Comment:
Any reason why this block of code was changed? This was done to better help tests use the resources generated by resource deployment. By default, resource deployment creates the AAD auth with the environment variables `<SDK group>_TENANT_ID`, etc and not the `Configuration` properties so that they won't potentially conflict.